ts/IterableInbox.tsx

```diff
@@ -22,46 +23,59 @@ const IterableInbox = ({
 }: inboxProps) => {
    const defaultInboxTitle = "Inbox"
    const [isDisplayMessage, setIsDisplayMessage] = useState<boolean>(false)
-   const [selectedMessageId, setSelectedMessageId] = useState<string>("")
-   const [messages, setMessages] = useState<InboxRowViewModel[]>([])
+   const [selectedMessageIdx, setSelectedMessageIdx] = useState<number>(0)
+   const [rowViewModels, setRowViewModels] = useState<InboxRowViewModel[]>([])
+
    const inboxDataModel = new IterableInboxDataModel()
+   // const inboxDeleteSource = new IterableInAppDeleteSource()
```

Comment:
We should delete irrelevant comments from the PR, they make reviewing more difficult than necessary.

Reply:
Agreed. Deleted all unnecessary comments.